components/pagination/src/PaginationItem.js

```diff
@@ -0,0 +1,90 @@
+/** @jsx jsx */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { jsx, useTheme } from '@westpac/core';
+import { SrOnly } from '@westpac/accessibility-helpers';
+
+// ==============================
+// Component
+// ==============================
+
+/**
+ * Pagination: Pagination is used to navigate within a set of related views. This element allows navigation within both finite (a set number) and infinite (unknown number) of views.
+ */
+export const PaginationItem = ({ first, last, active, disabled, children, ...props }) => {
+	const { COLORS } = useTheme();
+
+	return (
+		<>
+			<li
+				css={{
+					'a, span': {
+						padding: '0.4375rem 0.75rem',
+						border: `1px solid ${COLORS.border}`,
+						marginLeft: -1,
+						fontSize: '0.875rem',
+						color: active ? '#fff' : COLORS.neutral,
+						zIndex: active && 2,
+						backgroundColor: active && COLORS.hero,
+
+						...(first && {
+							marginLeft: 0,
+							borderTopLeftRadius: '3px',
+							borderBottomLeftRadius: '3px',
```

Comment:
These px values should be in rems. We made the decision that only border width is in px.